Frameworks/SPMySQLFramework/TimeZone.m

```diff
@@ -0,0 +1,50 @@
+//
+//  TimeZone.m
+//  SPMySQL.framework
+//
+//  Created by Robin Kunde on 8/19/20.
+//
+
+#import "TimeZone.h"
+#import "SPMySQLStringAdditions.h"
+
+@implementation SPMySQLConnection (TimeZone)
```

Comment:
Nit: This whole class feels like great adept for Swift implementation as we are adding it as a new class if you know Swift :)

Reply:
I don't think any of the existing Objc code even uses ARC, so we'd have to convert it before we can start mixing Objc and Swift code I think.